Ro-1439 Fjernet gammel KdvService og HelpTextService

gruble opened the pull request:
Testet på web og Android.

gruble committed:
Fjernet gammel KdvService basert på NanoSQL. Nå bruker vi KdvService fra common-modulen i stedet.

gruble committed:
Fjernet gammel KdvService basert på NanoSQL. Nå bruker vi KdvService fra common-modulen i stedet.

gruble committed:
lint fiks

gruble committed:
lint fiks

gruble committed:
Fjernet gammel HelpTextService og erstattet denne med HelpTextService fra common-modulen

gruble committed:
Fjernet gammel HelpTextService og erstattet denne med HelpTextService fra common-modulen

jorgkv commented on src/app/modules/registration/components/help-text/help-text.component.html:
```diff
@@ -1,7 +1,7 @@
   <ion-row>
     <ion-col class="ion-text-center">
-      <ion-button (click)="showHelp()" fill="clear">{{'HELP.TITLE' | translate}}</ion-button>
+      <ion-button (click)="showHelp(helpText)" fill="clear">{{'HELP.TITLE' | translate}}</ion-button>
```
Jeg tenker at det kanskje er bedre å hente helptextene inne i `showHelp`-funksjonen med feks `firstValueFrom`. I mange tilfeller vil ingen trykke på hjelpetekst-knappen i det hele tatt, og da er det litt unødvendig å hente hjelpetekstene hver eneste gang, sånn som det er satt opp her?

jorgkv reviewed (changes requested):
Ser bra ut! 👍 
Har et lite endringsforslag bare.

gruble committed:
Merge branch 'develop' into feature/ro-1439-remove-old-kdv-service

gruble committed:
Hjelpeteksten blir nå lastet kun når man trykker på hjelpeknappen

gruble committed:
Hjelpeteksten blir nå lastet kun når man trykker på hjelpeknappen

gruble commented:
> Ser bra ut! 👍 Har et lite endringsforslag bare.

Takk for innspill!
Har forsøkt å endre det nå slik at vi ikke formidler hjelpeteksten via html-malen. Tror fortsatt vi må finne hjelpeteksten for å sjekke at vi har en hjelpetekst, siden det er dette som bestemmer om knappen skal vises. Jeg tok tida, og ser ikke ut som vi sparte noe tid på å gjøre det på den nye måten, men det ble kanskje litt ryddigere.

gruble committed:
RO-1638 og RO-1643: Byttet 'dirt' til 'soil' et par steder siden API v.5 bruker 'soil' (#208)

* Byttet 'dirt' til 'soil' et par steder siden API v.5 bruker 'soil'

* Fiksa en import som var utdatert

jorgkv committed:
Fix RO-1590 unstable tests (#210)

* Fix test causing no window to close err

* Modify ci pipeline

  - Use npm cache
  - Use npm ci
  - Skip build step
  - Always try to publish test results

gruble committed:
Merge branch 'feature/ro-1439-remove-old-kdv-service' of https://github.com/NVE/regObs4 into feature/ro-1439-remove-old-kdv-service
